.changeset/purple-rings-swim.md

```
@@ -0,0 +1,6 @@
+---
+"ensindexer": minor
+"ensadmin": minor
+---
+
+Stopped having dependency on `@ensnode/ponder-metadata` package.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)